tests/conftest.py

```diff
@@ -73,3 +86,20 @@ def main_page_car_hire_tab(browser):
 @pytest.fixture(scope="module")
 def main_page_search_hotels_tab(browser):
     return MainPageSearchHotelsTab(browser=browser, url=browser.current_url)
+
+
+@pytest.fixture(scope="module")
+def assertions(browser):
+    return Assertions(browser=browser)
+
+
+@pytest.fixture()
+def user(request):
+    user_cmd = request.config.getoption("user")
+    return user_cmd
+
```

Comment:
if user_cmd == "None":
        return config.DefaultCreds.USERNAME
    else:
        return user_cmd
I consider add this

Reply:
Done